cmd/monaco/download/download_configs_test.go

```diff
@@ -316,34 +316,6 @@ func Test_shouldDownloadSettings(t *testing.T) {
 	}
 }
 
-func TestDownloadConfigsExitsEarlyForUnknownAPI(t *testing.T) {
```

Comment:
I might be missing it, but is there a new test covering the functionality now?
We still fail on unknown APIs and IMO it's worth having a test at this level - after all we require known APIs from this level down.

Reply:
added a new test with the name `Test_downloadConfigsOptions_valid`